src/Time/Actions/SelfStartActions.hpp

```diff
@@ -323,66 +343,6 @@ struct CheckForOrderIncrease {
   }
 };
 
-/// \ingroup ActionsGroup
-/// \ingroup TimeGroup
-/// Jumps to the start of the self-start algorithm (skipping taking a
-/// step from the last point) if the generation of the points for the
-/// current order is complete.
-///
-/// Uses:
-/// - GlobalCache: nothing
-/// - DataBox:
-///   - Tags::Next<Tags::TimeStepId>
-///   - SelfStart::Tags::InitialValue<variables_tag>
-///   - SelfStart::Tags::InitialValue<primitive_variables_tag> if the system
-///     has primitives
-///
-/// DataBox changes:
-/// - Adds: nothing
-/// - Removes: nothing
-/// - Modifies:
-///   - variables_tag if there is an order increase
-///   - primitive_variables_tag if there is an order increase and the system
-///     has primitives
-template <typename RestartTag>
-struct StartNextOrderIfReady {
```

Comment:
Just so I understand properly -- is the reason that this was required previously and isn't required anymore because the self-start procedure needs to skip the 'last' cleanup that would otherwise happen following the final update_u before starting the next order?